docs/using-airbyte/getting-started/oss-quickstart.md

```diff
@@ -154,24 +160,26 @@ abctl local install
 
 Your browser may open automatically to the Airbyte Application. If not, access it by visiting [http://localhost:8000](http://localhost:8000).
 
-When prompted for a username and password, enter the following default values: 
-- username: `airbyte`
-- password: `password`
+You will be asked to enter your email address and an organization name. Your email address will be used to authenticate
+to your instance of Airbyte. You will also need a password, to get your password run:
```

Comment:
Maybe throw in a quick blurb about how the password is generated?
e.g.
`You will also need a password, which is randomly generated as part of the install command. To get your password run...`